src/applications/vaos/containers/VAOSApp.jsx

```diff
@@ -56,12 +56,12 @@ export class VAOSApp extends React.Component {
 
     return (
       <RequiredLoginView
-        authRequired={1}
         serviceRequired={[
           backendServices.USER_PROFILE,
           backendServices.FACILITIES,
         ]}
         user={user}
+        verify
```

Comment:
This triggers the verification logic in this component, which will check the MVI profile status, which should correlate with the user having an ICN or not.